### ISSUE TITLE
Internal ADE Accessibility Short Ticket - TalkBack doesn't announce the selected "To (Required)" recipient, unlabeled close and clear field buttons

### ISSUE THREAD

**Issue by mmutia** [department-of-veterans-affairs/va-mobile-app#11457]
> Hi @BeccaTupaj and @digitalken,
> 
> I have confirmed that the bug also occurs on Android as well! Upon choosing a recipient from the "To" list, the selection cannot announced via TalkBack.
> 
> Additionally, the following buttons are unlabeled and require descriptive labels to inform the user of its purpose:
> 
> - the button to clear the "To" field
> - the button to close the "Search for a VA health care team" modal
> 
> CC @artsymartha68  

 _Originally posted by @mmutia in [#11453](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/11453#issuecomment-3180664868)_

**Comment by artsymartha68** [comment]
OK I found how to transfer it - hopefully I didn't break anything.

**Comment by digitalken** [comment]
@BeccaTupaj - I've reproduced the bug on iOS and Android.   Since the user has no idea who they sent the message to, I think I might give this a sev-2.

In iOS "com-bah" box with accent on "bah" does sound odd.

The designs for the "To:" field don't match the Pick a care system or Category designs.  Is that because of the search component?

Buttons:

- button to clear the "To" field:  iOS and Pixel do not describe the "x" function. Samsung does.
- button to close the "search for a VA health care team" modal: Samsung says "button, close, detected icon close, double tap to activate". iOS says "Close, button, close".  Pixel says "button, double tap to activate".  Need better labels.

**Comment by BeccaTupaj** [comment]
This bug will be resolved with ticket #11453
